### PR TITLE
Fix some System.Net.WebSockets.Client.Tests failures

### PR DESCRIPTION
**Pull request by schen381**
Contributes to: #19967

### CODE REVIEW

**Comment by schen381** [comment]
@dotnet-bot help

**Comment by schen381** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug
@dotnet-bot test outerloop netcoreapp Windows_NT Release
@dotnet-bot test outerloop netfx Windows_NT Debug
@dotnet-bot test outerloop netfx Windows_NT Release

@dotnet-bot test outerloop netcoreapp Ubuntu16.10 Release
@dotnet-bot test outerloop netcoreapp OSX10.12 Debug

**Comment by schen381** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug
@dotnet-bot test outerloop netcoreapp Windows_NT Release
@dotnet-bot test outerloop netfx Windows_NT Debug
@dotnet-bot test outerloop netfx Windows_NT Release

@dotnet-bot test outerloop netcoreapp Ubuntu16.10 Release
@dotnet-bot test outerloop netcoreapp OSX10.12 Debug

File: src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public static void UseDefaultCredentials_Roundtrips()
             Assert.False(cws.Options.UseDefaultCredentials);
         }
 
-        [ActiveIssue(19967, TargetFrameworkMonikers.NetFramework)]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/20358", TargetFrameworkMonikers.NetFramework)]
```
Comment (CIPop) [inline src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs]: We've decided to use numbers only.

Comment (schen381) [inline src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs]: ok

**Comment by schen381** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug
@dotnet-bot test outerloop netcoreapp Windows_NT Release
@dotnet-bot test outerloop netfx Windows_NT Debug
@dotnet-bot test outerloop netfx Windows_NT Release

@dotnet-bot test outerloop netcoreapp Ubuntu16.10 Release
@dotnet-bot test outerloop netcoreapp OSX10.12 Debug

**Comment by danmosemsft** [comment]
@odtnet-bot test Linux x64 Tests - Debug - Ubuntu.1604.Amd64.Open 
 (https://github.com/dotnet/corefx/issues/20245)

**Comment by danmosemsft** [comment]
@dotnet-bot test Linux x64 Tests - Debug - Ubuntu.1604.Amd64.Open 
 (https://github.com/dotnet/corefx/issues/20245)

**Comment by danmosemsft** [comment]
@dotnet-bot test OuterLoop netcoreapp Windows_NT Debug x64 (test fix)
@dotnet-bot test OuterLoop netcoreapp Windows_NT Release x64 (test fix)

**Comment by schen381** [comment]
@dotnet-bot test outerloop netfx Windows_NT Debug
@dotnet-bot test outerloop netfx Windows_NT Release
@dotnet-bot test Linux x64 Tests - Debug - Ubuntu.1604.Amd64.Open
@dotnet-bot test Linux x64 Tests - Release - Ubuntu.1404.Amd64.Open
@dotnet-bot  tes Innerloop netfx Windows_NT Release x64 Build and Test

**Comment by schen381** [comment]
@dotnet-bot test Innerloop netfx Windows_NT Release x64 Build and Test
@dotnet-bot test Linux x64 Tests - Debug - Ubuntu.1604.Amd64.Open
@dotnet-bot test Linux x64 Tests - Release - Ubuntu.1404.Amd64.Open

**Comment by schen381** [comment]
@dotnet-bot test Linux x64 Tests - Debug - Ubuntu.1604.Amd64.Open
@dotnet-bot test Linux x64 Tests - Release - Ubuntu.1404.Amd64.Open
@dotnet-bot test OuterLoop netcoreapp Windows_NT Debug x64
@dotnet-bot test OuterLoop netcoreapp Windows_NT Release x64
@dotnet-bot test OuterLoop netfx Windows_NT Debug x64 
@dotnet-bot test OuterLoop netfx Windows_NT Release x64

File: src/System.Net.WebSockets.Client/tests/CancelTest.cs [hunk 2]
```diff
@@ -163,6 +162,7 @@ public class CancelTest : ClientWebSocketTestBase
                 
                 Task recieve = cws.ReceiveAsync(segment, cts.Token);
                 cts.Cancel();
+                await Assert.ThrowsAnyAsync<OperationCanceledException>(() => recieve);
```
Comment (davidsh) [inline src/System.Net.WebSockets.Client/tests/CancelTest.cs]: "recieve" should be spelled as "receive".

**Comment by schen381** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug
@dotnet-bot test outerloop netfx Windows_NT Debug
@dotnet-bot test outerloop netcoreapp OSX10.12 Debug

**Comment by schen381** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug
@dotnet-bot test outerloop netfx Windows_NT Debug

Both:
There were xunit test failures in System.Diagnostics.Process.Tests
System.Diagnostics.Tests.ProcessStartInfoTests.StartInfo_TextFile_ShellExecute [FAIL]

**Comment by davidsh** [comment]
Unrelated CI failures in NETFX Outerloop:
https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_debug_prtest/34/

Merging since everything else is green.

**Comment by stephentoub** [comment]
@davidsh, @schen381, does some or all of this change need to be ported to release/2.0?  If so, please follow-up accordingly. Thanks.

**Pull request merged**